eng/Packages.Data.props

```diff
@@ -57,6 +57,28 @@
   <!--
     Dependency versions for Track 2, Azure.* libraries.
     Only packages that are approved dependencies should go here.
```

Comment:
Should we link to, or mention, the process for getting approved?

Reply:
Would this work? https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-things-to-know I couldn't find anything better

Reply:
I'd vote for just grabbing the email address and mentioning that if you need approval, that's who you talk to.   I'm afraid that it gets lost in all the words in the contributing guide.

Reply:
updated